src/vs/workbench/contrib/chat/browser/actions/chatAccessibilityHelp.ts

```diff
@@ -56,6 +57,8 @@ function descriptionForCommand(commandId: string, msg: string, noKbMsg: string,
 
 export async function runAccessibilityHelpAction(accessor: ServicesAccessor, editor: ICodeEditor, type: 'chat' | 'editor'): Promise<void> {
 	const widgetService = accessor.get(IChatWidgetService);
+	const contextKeyService = accessor.get(IContextKeyService);
+	const contextKey = type === 'editor' ? CTX_INTERACTIVE_EDITOR_ACCESSIBILTY_HELP_VISIBLE.bindTo(contextKeyService) : undefined;
```

Comment:
I think we are reaching a point at which we should inverse control. Instead of putting more editor vs chat knowledge into this we should give more responsibility to the caller. I suggest

* add `InteractiveEditorController#showAccessibilityHelp`
* the controller owns the context key and updates it accordingly in `showAccess....`
* add `hideAccessibilityHelp` as a counterpart to show

Not yet sure if this should be a separate state or just a mode of the widget (I think the latter tho). 

Having this would also allow us have a better messages, e.g hardcoding /explain and /fix as it is done today isn't correct. Those commands are defined by extensions and we cannot statically know what they do or if they exist.

Reply:
that sounds good for next iteration and is in alignment with https://github.com/microsoft/vscode/issues/183363

Reply:
Both there is also proper way to achieve this. The `/explain` slash command uses an [API flag](https://github.com/microsoft/vscode/blob/670f92f22b3e2ce78964ba12326ccd56ddd0ebfb/src/vs/workbench/contrib/interactiveEditor/common/interactiveEditor.ts#L24) to indicate that this will happen. Instead of hardcoding we should read that property (of all slash commands) and craft the message accordingly. And/or we read this when IntelliSense shows/completes these commands

Reply:
I have no idea how to get that from the editor without refactoring all of this code, so will have to wait until June